db/migrate/20191124212459_trades_foreign_keys.rb

```diff
@@ -1,6 +1,6 @@
 class TradesForeignKeys < ActiveRecord::Migration[5.2]
   def change
-  	add_foreign_key :trades, :orders, column: :buy_order_id, primary_key: "id"
-    add_foreign_key :trades, :orders, column: :sell_order_id, primary_key: "id"
+  	#add_foreign_key :trades, :orders, column: :buy_order_id, primary_key: "id"
```

Comment:
I would suggest just deleting the migration file since this migration file isn't doing anything (#19).